docs/deployment-options.md

```
@@ -0,0 +1,650 @@
+s3cmd get s3://your-space/backups/sanctum-YYYYMMDD.tar.gz /tmp/
+tar -xzf /tmp/sanctum-YYYYMMDD.tar.gz -C /
+
+# Start services
+docker compose -f docker-compose.prod.yml up -d
+```
+
+---
+
+## Amazon Web Services (AWS)
+
+*Coming soon*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)